### PR TITLE
K8S-2189: Operational diagram for Couchbase Service Broker

### PR DESCRIPTION
**Pull request by eric-schneider**
[K8S-2189](https://issues.couchbase.com/browse/K8S-2189): Create operational diagram for Couchbase Service Broker

This change adds an operational diagram to the CSB docs home. This is part of the overall Cloud-Native docs homepage enhancements requested by PM for the 2.2 release [K8S-2157](https://issues.couchbase.com/browse/K8S-2157).

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -29,15 +29,35 @@ This leads to some key benefits:
...
 
 Resources are templated, so they may be dynamically configured based on a set of inputs.
 These inputs include metadata--provided by the client to provision the instance--and user specified parameters.
 
+image::csb-overview-diagram.png[]
+
+. Let’s say we have a Service Broker created and deployed on the Kubernetes platform.
+
+. Register the Service Broker with the platform.
+Once the Service Broker is registered with the platform (Kubernetes), users can view the service offerings by listing all the services and service plans provided by different Service Brokers through the service catalog.
```
Comment (spjmurray) [inline modules/ROOT/pages/index.adoc]: It's registered with the service catalog, which is just another operator, nothing to do with the platform.  Unless platform is the service catalog platform and not the kubernetes platform... or the platform is an aggregate of kubernetes and the service catalog.

**Review by spjmurray — COMMENTED** [review]
I'm... okay... I guess, the diagram kinda makes sense, but it's just a spiders web with no real flow.  Some technical stuff I feel uncomfortable with depending on how you interpret it.

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (eric-schneider) [reply modules/ROOT/pages/index.adoc]: I'll get your diagram in there some way or another. Promise

**Review by spjmurray — APPROVED** [review]
Good enough I guess for the audience.

**Pull request merged**